### PR TITLE
Feb '24  OTP-UI Update

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**

Vehicle rental overlay / OTP2 layer missing TODO

https://github.com/opentripplanner/otp-ui/pull/715
https://github.com/opentripplanner/otp-ui/pull/716

**PR Checklist:**
- [x] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [x] Are all languages supported (Internationalization/Localization)?
- [x] Are appropriate Typescript types implemented?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,29 +42,29 @@
...
-    "@opentripplanner/base-map": "^3.0.16",
-    "@opentripplanner/core-utils": "^11.2.3",
-    "@opentripplanner/endpoints-overlay": "^2.0.11",
-    "@opentripplanner/from-to-location-picker": "^2.1.11",
-    "@opentripplanner/geocoder": "^2.0.1",
+    "@opentripplanner/base-map": "^3.0.17",
+    "@opentripplanner/core-utils": "^11.3.0",
+    "@opentripplanner/endpoints-overlay": "^2.0.12",
+    "@opentripplanner/from-to-location-picker": "^2.1.12",
+    "@opentripplanner/geocoder": "^2.0.2",
```
Comment (binh-dam-ibigroup) [inline package.json:49]: Geocoder 2.1.0 will be available shortly

Comment (miles-grant-ibigroup) [reply package.json:49]: will update when I update vehicle rental overlay

Comment (miles-grant-ibigroup) [reply package.json:49]: have assigned you the blocker for consistency

Comment (miles-grant-ibigroup) [reply package.json]: Updated!

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
Can some of the OTP-UI deps be deduplicated?

**Review by binh-dam-ibigroup — APPROVED** [review]

**Comment by daniel-heppner-ibigroup** [comment]
Nice, thanks for the update!

**Review by daniel-heppner-ibigroup — APPROVED** [review]

**Pull request merged**